task2.py

```diff
@@ -0,0 +1,73 @@
+# Person names
+names = [
+    "Smith",
+    "Johnson",
+    "Williams",
+    "Jones",
+    "Brown",
+    "Davis",
+    "Miller",
+    "Wilson",
+    "Moore",
+    "Taylor",
+    "Anderson",
+    "Thomas",
+    "Jackson",
+    "White",
+]
+#Create dictionary name: name length
+names_length = {name: len(name) for name in names}
+print("Length of names", list(names_length.values()))
+
+# Sorting 
+sorted_names_length = sorted(names_length.items(), key=lambda x: x[1])
+
+
+#Create dictionary name length vise value
+length_wise_names = dict()
+for name_length_tuple in sorted_names_length:
+    if name_length_tuple[1] in length_wise_names.keys():
+        length_wise_names[name_length_tuple[1]].append(name_length_tuple[0])
+    else:
+        length_wise_names[name_length_tuple[1]] = [name_length_tuple[0]]
+
+
+# 3 most frequent names
+print("\n The three most frequent name lenghts are:")
+three_most_freq_names = sorted(length_wise_names)[:3]
```

Comment:
You don't have to sort again, because `lengths_wise_names` is already sorted. It is like `{5: [], 6: [], 8: [], 11: []}`. You can print and check that it is already sorted, then why are you sorting again?